cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -1270,6 +1271,8 @@ class StructureBoard {
         }
         var fixedContentMarkup = contentMarkup;
         var newDoc = new DOMParser().parseFromString(fixedContentMarkup, 'text/html');
+        let new_scripts = $(newDoc).find('script');
```

Comment:
this is a nitpick, but in cms js we generally try to avoid snake case in variable names

Reply:
Sorry for that, I'm a python guy, so snake case is my natural style, but I will change it 😃